UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.API/Controllers/EssWebhookController.cs

```diff
@@ -0,0 +1,71 @@
+﻿using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using UKHO.ExchangeSetService.API.Services;
+using UKHO.ExchangeSetService.Common.Logging;
+using UKHO.ExchangeSetService.Common.Models.Request;
+
+namespace UKHO.ExchangeSetService.API.Controllers
+{
+    [ApiController]
+    [Authorize]
+    public class EssWebhookController : BaseController<EssWebhookController>
+    {
+        private readonly IEssWebhookService essWebhookService;
+
+        public EssWebhookController(IHttpContextAccessor contextAccessor,
+                                    ILogger<EssWebhookController> logger,
+                                    IEssWebhookService essWebhookService)
+        : base(contextAccessor, logger)
+        {
+            this.essWebhookService = essWebhookService;
+        }
+
+        [HttpOptions]
+        [Route("/Options")]
+        public IActionResult Options()
+        {
+            using (var reader = new StreamReader(Request.Body, Encoding.UTF8))
+            {
+                var webhookRequestOrigin = HttpContext.Request.Headers["WebHook-Request-Origin"].FirstOrDefault();
+                ////var webhookRequestCallback = HttpContext.Request.Headers["WebHook-Request-Callback"];
+                ////var webhookRequestRate = HttpContext.Request.Headers["WebHook-Request-Rate"];
+                HttpContext.Response.Headers.Add("WebHook-Allowed-Rate", "*");
+                HttpContext.Response.Headers.Add("WebHook-Allowed-Origin", webhookRequestOrigin);
+            }
+            return Ok();
+        }
+
+        [HttpPost]
+        [Route("/PostEssWebhook")]
```

Comment:
Can we use lowercase letters for route endpoints like [Route("/postEssWebhook")]